README.md

```diff
@@ -15,25 +17,31 @@ package main
 import (
     "fmt"
 
-    "github.com/tebeka/atexit"
+    "github.com/aureolebigben/atexit"
 )
 
 func handler() {
     fmt.Println("Exiting")
 }
 
+func fooHandler() {
+    fmt.Println("not called")
+}
+
 func main() {
-	atexit.Register(handler)
-	atexit.Exit(0)
+    _ = atexit.Register(handler)
```

Comment:
What is the value of `_ =`? How does it contribute?

Reply:
i added `_ =` because it's a best practice. It's use to say that we don't want the returned value